### PR TITLE
Fix/filter out disabled chains

### PR DESCRIPTION
**Pull request by svojsu**
- Filter enabled chains for some parts of the app;
- Remove `noFilter` for `ChainFilterStrategy`

### CODE REVIEW

File: novawallet/Common/Helpers/ChainsStore.swift [hunk 1]
```diff
@@ -51,6 +61,10 @@ extension ChainsStore: ChainsStoreProtocol {
     }
 
     func getChain(for chainId: ChainModel.Id) -> ChainModel? {
-        chainRegistry.getChain(for: chainId)
+        let chain = chainRegistry.getChain(for: chainId)
```
Comment (ERussel) [inline novawallet/Common/Helpers/ChainsStore.swift:64]: As we agreed let's also filter available chains to be consistent. So, I suggest filter availableChainIds applying filter to the corresponding chain (chainRegistry.getChain(...))

**Review by ERussel — COMMENTED** [review]
Don't see where we filtered chains for the cross chain transfers

**Comment by ERussel** [comment]
> Don't see where we filtered chains for the cross chain transfers

Nevermind, found changes

File: novawallet/Common/Helpers/ChainsStore.swift (hunk 1 above)
Comment (svojsu) [reply novawallet/Common/Helpers/ChainsStore.swift:77]: Thanks, done

**Review by ERussel — APPROVED** [review]

**Pull request merged**